lib/chef/resource/rhsm_register.rb

```diff
@@ -121,24 +121,30 @@ class RhsmRegister < Chef::Resource
       end
 
       action_class do
+        #
+        # @return [Symbol] dnf_package or yum_package depending on OS release
+        #
         def package_resource
           node["platform_version"].to_i >= 8 ? :dnf_package : :yum_package
         end
 
+        #
+        # @return [Boolean] is katello-ca-consumer installed
```

Comment:
Was this supposed to have the same description as the following method?

Reply:
fixed